.github/workflows/test.yaml

```diff
@@ -176,6 +176,11 @@ jobs:
           run: make docker-build-collector COLLECTOR_IMG=${COLLECTOR_IMG}
         - name: Build trivy-scanner
           run: make docker-build-trivy-scanner TRIVY_SCANNER_IMG=${SCANNER_IMG}
+        - name: Install kind v0.17.0
```

Comment:
isn't e2e framework going to override this?

Reply:
e2e framework installs kind if it's not already present